primitives/core/src/traits.rs

```diff
@@ -69,11 +66,60 @@ pub trait BareCryptoStore: Send + Sync {
 
 	/// Get the password for this store.
 	fn password(&self) -> Option<&str>;
-
+	/// Provided a list of (CryptoTypeId,[u8]) pairs, this would return
```

Comment:
First doc header, then description separated by empty line renders much better in html

```rust
/// List of supported keys.
/// 
/// Provided a list of (CryptoTypeId,[u8]) pairs, this would return 
/// a filtered list of public keys which are supported by the keystore.
```